rust-src/concordium_base/src/protocol_level_tokens/token_metadata_url.rs

```diff
@@ -1,11 +1,16 @@
-use crate::common::cbor::Bytes;
+use crate::common::cbor::{value, Bytes};
 use concordium_base_derive::{CborDeserialize, CborSerialize};
+use std::collections::HashMap;
 
 /// Metadata for a specific protocol level token
-#[derive(Debug, Clone, Eq, PartialEq, CborSerialize, CborDeserialize)]
+#[derive(Debug, Clone, PartialEq, CborSerialize, CborDeserialize)]
 pub struct MetadataUrl {
     /// A string field representing the URL
     pub url:              String,
     /// An optional sha256 checksum value tied to the content of the URL
     pub checksum_sha_256: Option<Bytes>,
+    /// Additional fields may be included for future extensibility, e.g. another
+    /// hash algorithm.
+    #[cbor(other)]
+    pub other:            HashMap<String, value::Value>,
```

Comment:
I would suggest we name this `additional` as that's the name we use in other libraries
```suggestion
    pub additional:       HashMap<String, value::Value>,
```